chatterbox/src/Auth/Register/index.js

```diff
@@ -40,15 +40,15 @@ class Register extends Component {
         // alert("stage "+curr)
         let canNext = this.state.canNext;
         if (canNext) {
-            if (this.state.stage == 1) {
+            if (this.state.stage === 1) {
```

Comment:
Why setstate

Reply:
I don''t know. This was not a component that I was working on

Reply:
Ok can you maybe talk to the person that did this?

Reply:
yes I will do that